### ISSUE TITLE
node dist/index.js (custom-server-typescript)

### ISSUE THREAD

**Issue by RayLuxembourg** [zeit/next.js#9201]
# Bug report
Error: Cannot find module '/Users/rayluxembourg/Desktop/Projects/someProj/dist/index.js
## Describe the bug
Suddenly when i build the project there is no index.js in dist folder.
next version 9.1.1
## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:
yarn build 
look inside the dist folder no index.js

## Expected behavior
looking at the custom server typescript repo
yarn start does node dist/index.js
currently in order to fix this i do nod dist/server/index.js

## Screenshots

```
NODE_ENV=production node dist/index.js
internal/modules/cjs/loader.js:626
    throw err;
    ^

Error: Cannot find module '/Users/rayluxembourg/Desktop/Projects/someProj/dist/index.js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:623:15)
    at Function.Module._load (internal/modules/cjs/loader.js:527:27)
    at Function.Module.runMain (internal/modules/cjs/loader.js:837:10)
    at internal/main/run_main_module.js:17:11 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []

```
## System information

- OS: osx
- Version of Next.js: 9.1.1

**Comment by jamesmosier** [comment]
I pulled down the `custom-server-typescript` example and if you just run `yarn start` and not `yarn build` beforehand that is the error you receive, because that file is generated during the build. So ensure you are building before starting.

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
